packages/apps-config/src/api/spec/chainx.ts

```diff
@@ -0,0 +1,446 @@
+/* eslint-disable header/header */
+/* eslint-disable sort-keys */
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
+export default {
+  Chain: {
```

Comment:
I would just suggest adding the `Address` and `LookupSource` types explicitly. The reason is that the defaults are changing on the API layer, see https://github.com/polkadot-js/api/releases/tag/v3.5.1

(One of these types https://polkadot.js.org/docs/api/FAQ#i-cannot-send-transactions-sending-yields-address-decoding-failures)

Reply:
Ok. Thanks for your advice! it has been repaired.

Reply:
No issues, just trying to make sure you don't have pain later :)